sapi/apache2handler/apache_config.c

```diff
@@ -17,6 +17,12 @@
 #define ZEND_INCLUDE_FULL_WINDOWS_HEADERS
 
 #include "php.h"
+#ifdef strcasecmp
+# undef strcasecmp
+#endif
+#ifdef strncasecmp
+# undef strncasecmp
+#endif
```

Comment:
Who/what is providing the conflicting declarations here?

Reply:
These come from apr_general.h, see https://ci.appveyor.com/project/php/php-src/builds/33115213/job/vp5168d5kcwo19te?fullLog=true#L3766 ff for details.

Reply:
A possible alternative would be to make our own declarations conditional, and include the apache headers first. But I think either way is fine.

Reply:
Then I think we should leave it as is. :)